lib/oneview-sdk/resource/api600/c7000/switch.rb

```diff
@@ -9,20 +9,13 @@
 # CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 
-require_relative '../../api300/c7000/switch'
+require_relative '../../api500/c7000/switch'
 
 module OneviewSDK
   module API600
     module C7000
       # Switch resource implementation for API 500 C7000
```

Comment:
comment needs to be rectified.its 600 API

Reply:
Changed the version